__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -172,7 +178,8 @@ describe('TESTING ChatSchema', () => {
         message: {
           created_at: 'createdAt',
           id: 'messageId',
-          updated_at: 'updatedAt'
+          updated_at: 'updatedAt',
+          attachments: '[]'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` field should be a string that contains serialized JSON.
```diff
-  attachments: '[]'
+  attachments: JSON.stringify([])
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->